### DISCUSSION TITLE
issue to upload large file

### DISCUSSION THREAD

**Discussion by Sayuksh** [community/community#70083] [New to GitHub]
I am trying to upload a 360 MB file and cannot do so.  I used git lfs and I took help with the YouTube tutorial I even tried GitHub desktop and still failed can anyone help?

**Comment by YurinDoctrine** [comment]
Uploading a large 360 MB file using Git or Git LFS can be challenging due to their limitations on file size. Git is not designed for handling large files efficiently, and Git LFS (Large File Storage) is a better solution for managing large files within a Git repository.

**Comment by NushaMBZ** [comment]
Having trouble uploading a large file to a Git repository is a common issue. Git and GitHub have limitations on the size of files that can be uploaded, and if your file is 360 MB, it may exceed those limits.
